lib/puppet/parser/functions/ip_to_cron.rb

```diff
@@ -0,0 +1,55 @@
+#
+# ip_to_cron.rb
+#
+# Copyright 2016 Vox Pupuli
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+module Puppet::Parser::Functions
+  newfunction(:ip_to_cron, :type => :rvalue, :doc => <<-EOS
+Returns an array of valid cron times given an optional runinterval
+
+Prototype:
+
+    ip_to_cron(n)
```

Comment:
And if you use this for many different cron jobs, they will all run at the same time... is that useful? Would it not be useful to say `command_runinterval_to_cronjob("echo my cron ran","3600")` and get back a full cron line, or ever `generate` a cron puppet resource.

I guess I just don't see the utility of this.